src/controllers/v1/availability.js

```diff
@@ -0,0 +1,106 @@
+// Dependencies
+const availabilityService = require('@services/availability')
+
+/**
+ * A class representing Availability and its related operations.
+ */
+module.exports = class Availability {
+	/**
+	 * Create availability.
+	 * @method
+	 * @name create
+	 * @param {Object} req - The request object containing the request body and decoded token.
+	 * @returns {Promise<Object>} - The created availability object.
+	 */
+	async create(req) {
```

Comment:
@nevil-mathew update the postman collection also

Reply:
updated